app/src/main/java/org/gnucash/android/service/ScheduledActionService.java

```diff
@@ -175,14 +175,32 @@ private static int executeBackup(ScheduledAction scheduledAction, SQLiteDatabase
         try {
             //wait for async task to finish before we proceed (we are holding a wake lock)
             new ExportAsyncTask(GnuCashApplication.getAppContext(), db).execute(params).get();
-            scheduledAction.setExecutionCount(++executionCount);
+            ++executionCount;
         } catch (InterruptedException | ExecutionException e) {
             Crashlytics.logException(e);
             Log.e(LOG_TAG, e.getMessage());
         }
+
+        // HACK: Add missed runs to executionCount so computeNextScheduledExecutionTime()
```

Comment:
If this fixes the problem, then we can live with it for now. As long as the meaning of the executionCount is not changed.
In the future backups & exports may be separated from scheduled transactions and then we will have to revisit this.